python/kfserving/docs/V1alpha2AlibiExplainerSpec.md

```diff
@@ -4,7 +4,7 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **config** | **dict(str, str)** | Inline custom parameter settings for explainer | [optional] 
-**resources** | [**V1ResourceRequirements**](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ResourceRequirements.md) | Defaults to requests and limits of 1CPU, 2Gb MEM. | [optional] 
+**resources** | [**V1ResourceRequirements**](V1ResourceRequirements.md) | Defaults to requests and limits of 1CPU, 2Gb MEM. | [optional] 
```

Comment:
add back the link? unfortunately we need to manually tweak this.

Reply:
how do we adjust? currently, I generate automatically.

Reply:
Can you help rebase from master, there are a few fixes from the v1beta1 SDK PR https://github.com/kubeflow/kfserving/pull/1150